### PR TITLE
added flextesa instructions

### LINKED ISSUE #2464: create details in integration test readme on using flextesa
As part of upgrading the integration test readme for Mumbai, we removed the flextesa information as it requires extensive revision. The revision will be done separately per this issue.

Please create a section for the integration test readme that explains how to use flextesa to run the integration tests.

### PR DESCRIPTION
**Pull request by michaelkernaghan**
following up a revision of the integration test readme by adding to it a revised instruction set for running integration tests with flextesa.

CLOSE #2464

### CODE REVIEW

File: integration-tests/README.md [hunk 1]
```diff
@@ -1,131 +1,224 @@
...
-The integration tests can be run in two environments: a local sandbox, and against a live Tezos testnet.
+## Running Integration Tests Against a Tezos Testnet
 
-## How to Run the Integration Tests Against a Local Sandbox
+To run tests in this environment, make sure you have:
 
-**Note: The recommended method to run tests is against testnets, not sandboxes. Running all tests against a sandbox can fail randomly, while running individual tests usually passes.**
+- Taquito source code cloned to your local machine
+- A compatible version of Node.js installed (see the top-level README.md)
+- Successfully compiled Taquito (see the top-level README)
```
Comment (AlirezaHaghshenas) [inline integration-tests/README.md:12]: Here, something is missing. Using the default settings, people outside ECAD cannot run tests. They need to change node URLs.

Comment (AlirezaHaghshenas) [reply integration-tests/README.md:12]: This has been missing, but it's now a good chance to fix it.

File: integration-tests/README.md [hunk 2]
```diff
@@ -1,131 +1,224 @@
...
-- Successfully compiled Taquito (see top-level README)
+To run tests against a node that is not preconfigured in Taquito, use:
 
-## Running all tests against all pre-configured testnets
+`export TEZOS_RPC_MUMBAINET='http://localhost:8732'`.
 
-From the `taquito/integration-tests` directory run
+## Using a Secret Key Instead of the Keygen API
+
+By default, the integration tests use an ephemeral key managed by the Keygen API. To use a secret key instead, use the CLI option `<testnet>-secret-key`, like this:
```
Comment (AlirezaHaghshenas) [inline integration-tests/README.md:66]: Also worth mentioning that this API is only available internally

**Review by AlirezaHaghshenas — APPROVED** [review]

File: integration-tests/README.md (hunk 1 above)
Comment (michaelkernaghan) [reply integration-tests/README.md:12]: I will add a third section to the readme to cover running the tets in parallel, how Taquito does it internally, and what external users can do,

File: integration-tests/README.md (hunk 2 above)
Comment (michaelkernaghan) [reply integration-tests/README.md:66]: done

**Pull request merged**